### ISSUE TITLE
Bale: Boat transport temporarily hangs under Indels Castle (1st screen after Dran)

### ISSUE THREAD

**Issue by DrewUniverse** [Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java#202]
```
12:05:03.457 [GPU legend.game.unpacker.Unpacker:66] INFO : Loading directory SECT/DRGN0.BIN/6789
12:05:03.521 [GPU legend.game.Scus94491BpeSegment:1925] INFO : Allocating script index 0 (0x0 bytes)
12:05:03.521 [GPU legend.game.Scus94491BpeSegment:2053] INFO : Loading script index 0 from 0x804f4c5c (entry point 0x0)
12:05:03.522 [GPU legend.game.Scus94491BpeSegment:1925] INFO : Allocating script index 10 (0x210 bytes)
12:05:03.522 [GPU legend.game.Scus94491BpeSegment:2053] INFO : Loading script index 10 from 0x804f40c8 (entry point 0x0)
12:05:03.523 [GPU legend.game.Scus94491BpeSegment_8002:370] INFO : Loading scripted TMD 804f07c4 (animation 804f00a0)
12:05:03.523 [GPU legend.game.Scus94491BpeSegment:1925] INFO : Allocating script index 11 (0x210 bytes)
12:05:03.523 [GPU legend.game.Scus94491BpeSegment:2053] INFO : Loading script index 11 from 0x804e8798 (entry point 0x0)
12:05:03.524 [GPU legend.game.Scus94491BpeSegment_8002:370] INFO : Loading scripted TMD 804e8288 (animation 804e823c)
12:05:03.524 [GPU legend.game.Scus94491BpeSegment:1925] INFO : Allocating script index 12 (0x210 bytes)
12:05:03.524 [GPU legend.game.Scus94491BpeSegment:2053] INFO : Loading script index 12 from 0x804e7e64 (entry point 0x0)
12:05:03.524 [GPU legend.game.Scus94491BpeSegment_8002:370] INFO : Loading scripted TMD 804e7474 (animation 804e7410)
12:05:03.527 [GPU legend.game.Scus94491BpeSegment:2118] INFO : Forking script 12 to 13
12:05:03.527 [GPU legend.game.Scus94491BpeSegment:1925] INFO : Allocating script index 13 (0x0 bytes)
12:05:03.589 [GPU legend.game.Scus94491BpeSegment_8002:370] INFO : Loading scripted TMD 804e4bf8 (animation 804e4bac)
```

**Comment by LordMonoxide** [comment]
I think this is likely due to collision issues. It looks like you're softlocked but it does warp ahead after a few seconds.

**Comment by LordMonoxide** [comment]
Fixed in 0370af1
